packages/nimble-components/src/table/template.ts

```diff
@@ -33,7 +36,8 @@ export const template = html<Table>`
             --ni-private-table-header-scrollbar-spacer-width: ${x => x.virtualizer.headerContainerMarginRight}px;
             --ni-private-table-scroll-height: ${x => x.virtualizer.allRowsHeight}px;
             --ni-private-table-row-container-top: ${x => x.virtualizer.rowContainerYOffset}px; 
-            --ni-private-table-row-grid-columns: ${x => x.rowGridColumns ?? ''}
+            --ni-private-table-row-grid-columns: ${x => x.rowGridColumns ?? ''};
+            --ni-private-collapse-button-visibility: ${x => (x.showCollapseAll ? 'visible' : 'hidden')};
```

Comment:
I think you can get rid of this `ni-private-collapse-button-visibility` CSS variable now.